xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -916,6 +916,12 @@ namespace PVR
     return true;
   }
 
+  bool CPVRGUIActions::CanEditRecording(const CFileItem &item) const
+  {
+    CGUIDialogPVRRecordingSettings RecordingSettings;
+    return RecordingSettings.CanEditRecording(item);
```

Comment:
As the function to call can be `statis` (see my other comment), this can be simplified to just one line:
`return CGUIDialogPVRRecordingSettings::CanEditRecording(item);`